src/index.js

```diff
@@ -37,16 +35,17 @@ const getDefaultOptions = () => ({
  * Method must return promise.
  * @ignore
  */
-const methodGroups = {
-    acts,
-    tasks,
-    keyValueStores,
-    datasets,
-    requestQueues,
-    logs,
-    users,
-    webhooks,
-    webhookDispatches,
+const endpointClasses = {
+    actors: Actors,
+    acts: Actors,
```

Comment:
Maybe we could add a decorator to `.acts.` methods that would print the deprecation warning.

Reply:
The warnings are already there.

Reply:
Warnings are there for old method names i.e.

```
client.acts.getAct();
client.actors.getAct();
```

but not for an old namespace with the new method name

```
client.acts.getActor();
```

Reply:
Hmm, good point! Although those methods never existed before (in old namespace) so it feels weird to deprecate them, since nobody could be using them.